policyengine_us/variables/gov/states/va/tax/income/deductions/cdcc_expense/va_cdcc_limit.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class va_cdcc_limit(Variable):
```

Comment:
actually i think we should still be clearer as to avoid misinterpretation as va having a cdcc. please update the label too
```suggestion
class va_child_dependent_care_deduction_cdcc_limit(Variable):
```